assets/src/edit-story/components/checklist/priorityChecks.js

```diff
@@ -40,6 +47,12 @@ export function PriorityChecks() {
         <StoryPosterAspectRatio />
         <PublisherLogoSize />
         <VideoElementMissingPoster />
+        {
+          // todo video optimization fails when auto optimization is turned off
```

Comment:
@swissspidy @spacedmonkey btw, we have been seeing an issue where if the auto optimize setting is turned off, the `optimizeVideo` promise errors out with a 500 (see @BrittanyIRL and @samwhale 's comments on this PR).

Reply:
This check is incorrect. Please copy these checks. 

https://github.com/google/web-stories-wp/blob/6972495f63e209e32d157ba6a9c283add1ec4eb5/assets/src/edit-story/app/media/utils/useMediaUploadQueue/index.js#L118-L121

Reply:
sorry guys. you're right. I remember this conversation i think during the bug bash where we originally started testing transcoding. I think the naming of this setting is causing some confusion. when I read the "auto optimization" setting details, I though that meant transcoding would not automatically happen when I drag and drop. Not meaning that it wouldn't be available at all.